crates/holochain/src/core/workflow/sys_validation_workflow/validation_query.rs

```diff
@@ -68,8 +98,8 @@ async fn get_ops_to_validate(
     sql.push_str(
         "
         ORDER BY
-        DhtOp.num_validation_attempts ASC,
-        DhtOp.op_order ASC
+        t.num_validation_attempts ASC,
```

Comment:
Please update the doc comment for this function regarding the ordering.

Reply:
Good spot. This function is actually called by both `get_ops_to_app_validate` and `get_ops_to_sys_validate`. Updated the doc comments on both to include the sorting by validation attempts.